pkgs/development/python-modules/django-ordered-model/default.nix

```
@@ -0,0 +1,56 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, isPy27
+
+# buildtime
+, setuptools
+
+# runtime
+, django
+
+# tests
+, python
+, djangorestframework
+}:
+
+buildPythonPackage rec {
+  pname = "django-ordered-model";
+  version = "3.7.4";
+  format = "pyproject";
+
+  disabled = isPy27;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-8li5diUlwApTAJ6C+Li/KjqjFei0U+KB6P27/iuMs7o=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+  ];
+
+  checkInputs = [
+    djangorestframework
+  ];
+
+  propagatedBuildInputs = [
+    django
+  ];
+
+  checkPhase = ''
+    runHook preCheck
+    ${python.interpreter} -m django test --settings tests.settings
+    runHook postCheck
+  '';
+
+  pythonImportsCheck = [ "ordered_model" ];
+
+  meta = with lib; {
+    description = "Allow models to be ordered and provide a simple admin interface for reordering them.";
```

```suggestion
    description = "Allow models to be ordered and provide a simple admin interface for reordering them";
```
Please remove the trailing period.
https://nixos.org/manual/nixpkgs/unstable/#var-meta-description